packages/editor/src/extensions/code-block/component.tsx

```diff
@@ -58,45 +58,57 @@ export function CodeblockComponent(
           overflow: "hidden"
         }}
       >
-        <Text
-          ref={(ref) => {
-            elementRef.current = ref ?? undefined;
-            forwardRef?.(ref);
+        <ScrollContainer
+          style={{
+            display: "flex",
+            overscrollBehavior: "contain"
           }}
-          autoCorrect="off"
-          autoCapitalize="none"
-          css={theme.codeBlockCSS}
-          sx={{
-            pre: {
-              fontFamily: "inherit !important",
-              tabSize: "inherit !important",
-              // background: "transparent !important",
-              padding: "10px !important",
-              margin: "0px !important",
-              width: "100%",
-              borderRadius: `0px !important`,
+          trackStyle={() => ({
+            backgroundColor: "transparent"
+          })}
+        >
+          <div style={{ flex: 1 }}>
```

Comment:
For some reason the editor hangs on page refresh if it contains a code block due to this. Haven't been able to figure out why though. Tested on Chrome.